README.md

```diff
@@ -57,11 +57,19 @@ for your specified platform. Additional, optional overrides can be provided:
     wait_for: [NUM OF SECONDS TO WAIT BEFORE TIMING OUT, DEFAULT 600]
     no_ssh_tcp_check: [DEFAULTS TO false, SKIPS TCP CHECK WHEN true]
     no_ssh_tcp_check_sleep: [NUM OF SECONDS TO SLEEP IF no_ssh_tcp_check IS SET]
-    networks: [LIST OF RACKSPACE NETWORK UUIDS, DEFAULT PUBLICNET AND SERVICE NET]
+    additional_networks: [LIST OF RACKSPACE NETWORK UUIDS, BESIDES PUBLICNET AND SERVICE NET]
     rackconnect_wait: ['true' IF USING RACKCONNECT TO WAIT FOR IT TO COMPLETE]
     servicelevel_wait: ['true' IF USING MANAGED SERVICE LEVEL AUTOMATION TO WAIT FOR IT TO COMPLETE]
     no_passwd_lock: ['true' IF FOG LIBRARY SHOULD NOT LOCK ROOT ACCOUNT]
-    servicenet: ['true' IF USING THE SERVICENET IP ADDRESS TO CONNECT]
+    servicenet: ['true' IF THE SERVICENET IP ADDRESS SHOULD BE USED TO CONNECT]
+    ssh_network_name: [IF SET, WILL USE THE SPECIFIED NETWORK TO CONNECT]
+    connect_public_net: [DEFAULTS TO true, SET TO false TO PROVISION NO PUBLIC IP]
```

Comment:
I started with that, but they're not equivalent.  if servicenet is true, then it uses the service net as the ssh network. If servicenet is false, it uses the publicnet (or the override ssh_network if one is specified) but it still connects the servicenet.  On the other hand, if publicnet is false, it doesn't even connect the public network.